lib/core/versions/latest/OperationProcessor.ts

```diff
@@ -71,13 +71,13 @@ export default class OperationProcessor implements IOperationProcessor {
     let encodedRevealValue;
     switch (operation.type) {
       case OperationType.Recover:
-        encodedRevealValue = (operation as RecoverOperation).recoveryRevealValue;
+        encodedRevealValue = (operation as RecoverOperation).signedData.recovery_reveal_value;
```

Comment:
My fault: do you think we should define the internal `signedDataModel` with camelCasing (only parser handles snake_casing validation and conversion in very isolated places) so we can get rid of this ugly snake_casing code creeping into places now? `CreateOperation` already does this for `SuffixDataModel`... again this was me letting this slipped through the cracks...